### ISSUE TITLE
Show only templates in 'Templates' - not all .html files

### ISSUE THREAD

**Issue by lydenyardley** [james2doyle/phileAdmin#14]
I have a load of html files inside my node_modules folder, and these all show up in the templates section of admin.

Can we work out a way around this? Maybe by only showing files in the root theme folder or by being able to add exceptions?

The first would be preferable I think.

**Comment by james2doyle** [comment]
What is the module you are using?

**Comment by lydenyardley** [comment]
Grunt

**Comment by james2doyle** [comment]
Ah ok I see them now. I was looking at pages. I think we might need to add a filter for ignoring certain folders in the themes directory.

**Comment by lydenyardley** [comment]
Wouldn't it work to just only show items that are in the root of the themes
directory?

**Comment by james2doyle** [comment]
No because there are folders like `partials` and such. I sometimes put more folders inside the partials folder if the site has a lot of different little pieces.

**Comment by khanduras** [comment]
I disagree, we should be able to see all html files. Perhaps a switch to toggle and hide, or even just a filter box to hide results you don't want to see.

**Comment by lydenyardley** [comment]
Yeah, I'm thinking that I wouldn't want a client seeing all these extra files...
